src/job.rs

```diff
@@ -1190,7 +1190,7 @@ fn perform_job_action(context: &Context, mut job: &mut Job, thread: Thread, trie
         Action::ImexImap => match JobImexImap(context, &job) {
             Ok(()) => Status::Finished(Ok(())),
             Err(err) => {
-                error!(context, "{}", err);
+                error!(context, "Import/export failed: {}", err);
```

Comment:
great to add some context here as well, this will help on fixing https://github.com/deltachat/deltachat-ios/issues/669 as it is one possible occurance. however, this has to remain an error!(), at least for now.